_layouts/chapter.html

```diff
@@ -0,0 +1,27 @@
+---
+layout: page
+---
+{{ content }}
+
+{% capture /dev/null %}
+  {% capture _previous_chapter %}[Table of contents](../){% endcapture %}
+  {% for chapter in site.data.scaling.toc %}
+    {% if _chapter_match %}
+      {% capture _next_chapter %}[{{chapter.name}}]({{chapter.permalink}}){% endcapture %}
+      {% break %}
+    {% endif %}
+    {% if chapter.permalink == page.url %}
+      {% assign _chapter_match = true %}
+    {% endif %}
+    {% unless forloop.first %}
+      {% capture _previous_chapter %}[{{chapter.name}}]({{chapter.permalink}}){% endcapture %}
```

Comment:
The previous chapter link seems to link to the current chapter for everything except the introduction.

Reply:
fixed